livekit-rtc/livekit/rtc/video_stream.py

```diff
@@ -31,16 +39,20 @@ def __init__(
         track: Track,
         loop: Optional[asyncio.AbstractEventLoop] = None,
         capacity: int = 0,
+        format: Optional[proto_video_frame.VideoBufferType.ValueType] = None,
```

Comment:
Can directly request a format from the VideoStream now

Reply:
This is great

Reply:
great! let's explain these concepts in the docstring please. that the native format will be returned.